main.go

```diff
@@ -33,6 +33,9 @@ type GAE struct {
 	// or to alter existing deployments.
 	// value will be sanitized (lowercase, replace non-alphanumeric with `-`, max 63 chars)
 	Version string `json:"version"`
+	// Service is used to set the service to be deployed
+	// value will be sanitized (lowercase, replace non-alphanumeric with `-`, max 63 chars)
```

Comment:
Just noticed this comment. I'm not seeing where we sanitize the value? If not, I'm fine with just dropping the comment?

Reply:
yes sorry this was a copy 🍝 from above and thought I had removed it